src/components/ProductsTable/ProductsTable.tsx

```diff
@@ -19,16 +20,22 @@ interface ProductsTableProps {
 }
 
 const ProductsTable: FunctionComponent<ProductsTableProps> = ({ data, isFetching }) => {
-  const columnNames = { name: 'Name', sku: 'SKU', quantity: 'Quantity' };
+  const columnNames = {
+    name: 'Name',
+    sku: 'SKU',
+    quantity: 'Quantity',
+    serviceLevel: 'Service Level',
+    unitOfMeasure: { name: 'Unit of measure' }
+  };
   const [page, setPage] = useState(1);
   const [perPage, setPerPage] = useState(10);
   const [searchValue, setSearchValue] = useState('');
   const [activeSortIndex, setActiveSortIndex] = React.useState<number>(0);
   const [activeSortDirection, setActiveSortDirection] = React.useState<'asc' | 'desc'>('asc');
 
-  const getSortableRowValues = (product: Product): (string | number)[] => {
-    const { name, sku, quantity } = product;
-    return [name, sku, quantity];
+  const getSortableRowValues = (product: Product): (string | number | UnitOfMeasure)[] => {
+    const { name, sku, quantity, serviceLevel, unitOfMeasure } = product;
```

Comment:
sorting of the unitOfMeasure column wasn't explicitly discussed, but what's here by default is off.
I think the sorting for this column should be grouped by name and then sorted by quantity (with unlimited being the highest).
it might get tricky with Not Available though because sorting by names (ascending) we'd have
Cores > Not Available > Nodes > Sockets

See if you can make it so Not Availble would be last in that list:
Cores > Nodes > Sockets > Not Available

Reply:
orrrr....we don't sort on Unit of Measure until we get someone from UX / Project to chime in.
in which case we can follow up with a story to do that later.

see if you can figure out the sorting i suggested, but if it's getting too hard just remove sorting on UoM and we'll come back later.